src/status_im2/contexts/communities/overview/utils.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im2.contexts.communities.overview.utils
+  (:require [clojure.string :as string]))
+
+(defn join-existing-users-string
+  [user-list]
+  (let [users-count (count user-list)
+        first-two   (->>
+                     user-list
+                     (take 2)
+                     (map #(first (string/split (:full-name %) #" "))))]
+    (case users-count
+      0 ""
+      1 (str "Join " (first first-two))
+      2 (str "Join " (string/join " and " first-two))
+      (str "Join " (string/join ", " first-two) " and " (- users-count 2) " more"))))
```

Comment:
It seems we should be using translations in this function.

Sorry to mention this if you already know, but you can create translations as templates, like:

```clojure
(i18n/label :t/change-log-level {:log-level name})
```

```json
"change-log-level": "Confirm and restart the app to change log level to {{log-level}}"
```